server/middlewares/fetch-user.js

```diff
@@ -42,7 +42,6 @@ function getEventsForUserId(client, user_id) {
               source,
               type
             } = e;
-            const { location = {} } = context;
```

Comment:
this is the syntax for making location an empty object if it doesn't exist in the context I guess?

Reply:
exactly :) https://wesbos.com/destructuring-default-values/